src/mo_frontend/parser.mly

```diff
@@ -733,6 +739,8 @@ exp_un(B) :
     { AndE(e1, e2) @? at $sloc }
   | e1=exp_bin(B) OR e2=exp_bin(ob)
     { OrE(e1, e2) @? at $sloc }
+  | e1=exp_bin(B) NULLCOALESCE e2=exp_bin(ob)
```

Comment:
Since this is a bit like a conditional, we could also consider allowing a block on the rhs (like an else branch).
```
| e1=exp_bin(B) NULLCOALESCE e2=exp_next
```
Maybe a bad idea

Reply:
Oh! Maybe allowing a block on the left? so that
```motoko
do ? {
} ?? default
```
Would parse properly without a parenthesis? 
Not allowing a block on the RHS would parse records better yes? Without extra brackets